api/test/gateways/UHW/UHWFetchDocuments.test.js

```diff
@@ -0,0 +1,66 @@
+const UHWFetchDocuments = require('../../../lib/gateways/UHW/UHWFetchDocuments');
+
+describe('UHWFetchDocumentsGateway', () => {
+  let buildDocument;
+  let db;
+
+  const createGateway = (records, throwsError) => {
+    buildDocument = jest.fn(({}) => {
```

Comment:
pretty sure we can simplify this to buildDocument = jest.fn(). what do you think?